sketch.js

```diff
@@ -263,6 +267,9 @@ function setup() {
 	let szofajok = ["főnév","határozó","ige","melléknév"];
 	createButtonBox(szofajok, maxWidth*2, x = width / 2, y = 150,75, szofajOnClick);
 
+	let saveFormats  = ["png","jpg"];
+	createButtonBox(saveFormats, maxWidth*2, x = width / 2, y = 700, 75, saveOnClick);
+
 }
 
 //TODO: use push and pop functionality better, wait for all pictures to load before drawing, because like this sometimes draws fail if we dont wait enough
```

Comment:
In draw function:
`	if(showText){
		text('Írj valamit:) Ha homályos nyomj egy entert', width/2 -250, height/15);
	  }`

Reply:
és a Word Drawjából szedjuk ki a textet.

Reply:
Megcsinálom akkor a megoldásod alapján. Lehet egységesebb, ha minden a canvason jelenik meg